moon/powerups/server/base.moon

```diff
@@ -1,7 +1,12 @@
-POWERUP_ID = "base-cfc-powerup"
-
 export BasePowerup
 class BasePowerup
+    @powerupList: {}
+
+    @powerupID: "base_cfc_powerup"
+    
+    @powerupTotalWeights: {0, 0, 0, 0}
+    @powerupWeights: {0, 0, 0, 0}
```

Comment:
I want to find a better way to format these. Maybe their keys can be like, `tier1`, `tier2` and so forth